### PR TITLE
Fix Faraday::TimeoutError on Chattermill Response Agent

### PR DESCRIPTION
**Pull request by monsha**
https://rollbar.com/chattermill/huginn/items/138

### CODE REVIEW

File: app/models/agents/chattermill_response_agent.rb [hunk 1]
```diff
@@ -193,129 +191,141 @@ def check
...
+        normalize = ->(name) {
           name.gsub(/(?:\A|(?<=-))([[:alpha:]])|([[:alpha:]]+)/) {
             $1 ? $1.upcase : $2.downcase
           }
         }
       when 'downcased'
         normalize = :downcase.to_proc
-      when 'snakecased', nil
-        normalize = ->name { name.tr('A-Z-', 'a-z_') }
+      when 'snakecased'
```
Comment (OpakAlex) [inline app/models/agents/chattermill_response_agent.rb:303]: what about `underscore`  ?

File: app/services/chattermill/response_parser.rb [hunk 2]
```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
```
Comment (OpakAlex) [inline app/services/chattermill/response_parser.rb:1]: What about some specs for this service?

**Comment by monsha** [comment]
@OpakAlex changes applied!

**Review by OpakAlex — APPROVED** [review]

**Review by duboff — APPROVED** [review]
LGTM

**Pull request merged**